### PR TITLE
create task4.py

### PR DESCRIPTION
**Pull request by anastasiarom**

### CODE REVIEW

File: Tasks/Romanchuk/Task4/task4.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+while True:
+    length = input('\nEnter the desired string length (must be >35): ')
+    if length.isdigit() and int(length) > 35:
+        length = int(length)
+        break
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Romanchuk/Task4/task4.py:5]: nice validation logic

File: Tasks/Romanchuk/Task4/task4.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    else:
+        print("\tInput Error: Impossible length value")
+
+with open("text.txt", 'r', encoding='utf-8') as text, open("formatted_text.txt", 'w') as new_text:
+    line = text.readline()
+    while line != '':
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Romanchuk/Task4/task4.py:11]: it can be just **while line** btw

**Review by MikitaTsiarentsyeu — APPROVED** [review]
just great, I can see that you look through some documentation, or ae able to google very well, either way the solution is one of the shortest possible, you did very good, and the score is 100 + bonus 10

**Comment by anastasiarom** [comment]
Thanks. But I haven't looked at any documentation and haven't googled. I wrote a lot of algorithms at the university in c/c++, and we couldn't use auxiliary libraries (for example to work with strings, lists, vectors and so on) and had to write all functions and classes ourselves. Therefore, now I have a habit of trying to optimize everything as much as possible in order to save time and space.

**Pull request merged**